src/labels/CustomLabels.labels

```diff
@@ -9875,6 +9875,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Error shown when errors must be removed before something else can occur.</shortDescription>
         <value>Clear errors before proceeding.</value>
     </labels>
+    <labels>
+        <fullName>stgConcurrentBatchError</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgConcurrentBatchError</shortDescription>
```

Comment:
@dcaballeroSf For the `shortDescription` on the new labels, can you put a description of the use of the label rather than just repeat the api name of the label? The `shortDescription` is an internal only comment that helps us understand the usage of our labels.